server/middleware/get-syndication-licence-for-user.js

```diff
@@ -54,6 +56,11 @@ module.exports = exports = async (req, res, next) => {
 
 		res.locals.licence = syndicationLicence;
 
+		if(!res.locals.hasGraphicSyndication) {
+			res.locals.hasGraphicSyndication = hasRichArticleAccess(syndicationLicence.products);
+		}
```

Comment:
We'd need a test for this middleware to make sure we set `hasGraphicSyndication` in req.locals

Reply:
👍